.circleci/config.yml

```
@@ -1353,6 +1418,12 @@ workflows:
                   enterprise_edition: true
                   context: cicd-orchestrator
                   name: Build and push docker images for APIM EE << pipeline.parameters.graviteeio_version >><<# pipeline.parameters.dry_run >> - Dry Run<</ pipeline.parameters.dry_run >>
+            -  publish_rpm_packages:
+                  context: cicd-orchestrator
+                  graviteeio_version: << pipeline.parameters.graviteeio_version >>
+                  dry_run: << pipeline.parameters.dry_run >>
+                  name: Publishing rpm packages
```

Let's follow the same convention as the other ones
```suggestion
                  name: Publishing RPM packages <<# pipeline.parameters.dry_run >> - Dry Run<</ pipeline.parameters.dry_run >>
```

---

done